debezium-core/src/main/java/io/debezium/relational/ddl/DataTypeBuilder.java

```diff
@@ -105,19 +106,30 @@ else if (parameters != null) {
             name.append(' ');
             name.append(suffix);
         }
-        return new DataType(cleanSignedUnsigned(expression), cleanSignedUnsigned(name), jdbcType, length, scale, arrayDims, arrayDimsLength);
+        return new DataType(cleanSignedUnsignedZerofill(expression), cleanSignedUnsignedZerofill(name), jdbcType, length, scale, arrayDims, arrayDimsLength);
     }
 
     /**
-     * Replace "signed unsigned" to "unsigned" when ddl statements contain multiple signed/unsigned keywords
+     * Clean the suffix name of numeric data type.
```

Comment:
I feel the JavaDoc needs to be expanded to describe the rules and reasons in more detail for future maintainer.

Reply:
done.